src/catalogue/Dockerfile

```diff
@@ -19,29 +21,35 @@ COPY *.go /go/src/mushop/catalogue/
 COPY go.mod /go/src/mushop/catalogue/
 
 # Build Catalogue service
-RUN GOARCH=amd64 GOOS=linux \
+RUN GOARCH=${ARCH} GOOS=${OS} \
   go build -a \
-  -o /app mushop/catalogue/cmd/cataloguesvc
+  -o /catalogue mushop/catalogue/cmd/cataloguesvc
 
 ##### Base image with Oracle Instant Client Basic Lite
-FROM oraclelinux:7-slim AS base
+FROM oraclelinux:8-slim AS base
 ARG clientVersion
-RUN yum update -y && \
-  yum -y install oracle-release-el7 && \
-  yum-config-manager --enable ol7_oracle_instantclient && \
-  yum -y install oracle-instantclient${clientVersion}-basiclite && \
-  yum clean all && \
-  rm -rf /var/cache/yum
+RUN microdnf update && \
+    microdnf install oracle-instantclient-release-el8 && \
+    microdnf install oracle-instantclient-basiclite && \
+    microdnf clean all && \
+    rm -rf /var/cache/dnf && \
+    rm -rf /var/cache/yum
 
 ##### Catalogue Service Image
 FROM base
 ARG clientVersion
 WORKDIR /
-COPY --from=go-builder /app /app
+COPY --from=go-builder /catalogue /catalogue
```

Comment:
reason for the switch ~`/app`~ to `/catalogue`?

Reply:
No special reason, just matching the build of the basic.https://github.com/oracle-quickstart/oci-cloudnative/blob/a927f28e244f324ee76b468b295438c9ebe30552/deploy/basic/Dockerfile#L115

Will also help when changing the dbtools strategy.

Reply:
I see. IIRC the reason of `basic` doing this is to differentiate from the other services built within the same Dockerfile. 🤷‍♂️

Reply:
I can revert to the app be called app again for the app binary run as `app`. Didn't know that you loved so much.

Reply:
Don't ❤️ . Just wanted to understand the reason - even if it's just a "feels good" 😄 . I'm ok as-is